src/services/strike_service.py

```diff
@@ -57,7 +57,7 @@ async def add_strike(
     log_info_and_add_field(
         logging_embed,
         logger,
-        "Result",
+        "Strike Points",
```

Comment:
i think it would be more useful to rename this as "Strike Point Adjustment" or something